src/index.js

```diff
@@ -105,10 +105,20 @@ export default class Undo {
     this.onUpdate();
   }
 
+  /**
+   * returns true if readOnly was toggled to true
+   * @returns {Node} Indirectly shows if readOnly was set to true or false
+   */
+  setReadOnly() {
+    const toolbox = document.querySelector('.ce-toolbox');
+    this.readOnly = !toolbox;
+  }
```

Comment:
@MarioRodriguezS does this change require some test scenarios?

Reply:
@juanhurtado10 At this moment, there is a test to the readOnly mode consistent with the new feature, however we could test specifically the new feature. Thanks for the comment!